libs/sdk/eslint-config/package.json

```diff
@@ -28,13 +28,8 @@
   },
   "peerDependencies": {
     "@angular/cli": "^15.2.8",
-    "@angular-eslint/eslint-plugin": "^15.2.1",
-    "@angular-eslint/eslint-plugin-template": "^15.2.1",
-    "@angular-eslint/template-parser": "^15.2.1",
-    "@typescript-eslint/eslint-plugin": "^5.59.5",
-    "@typescript-eslint/parser": "^5.59.5",
-    "eslint-plugin-deprecation": "^1.4.1",
-    "eslint": "^8.36.0"
```

Comment:
These peers are installed by the `@angular-eslint/schematics` package, so I'm reviewing them to avoid version conflicts.